org-mcb/config/issues-are-cached-and-incomplete/config.toml

```diff
@@ -0,0 +1,12 @@
+# This config is optimised for speed of local interation of general Hugo set-up without worrying about issue freshness or completeness.
+# To build with this config, run `hugo server --environment issues-are-cached-and-incomplete`.
+
+[params]
+# Don't use the paginating proxy, because it's slow.
+# This means some issues will be missing when fetched.
+issuesorgapi = "https://api.github.com/repos/Migracode-Barcelona/"
```

Comment:
we can probably rm all this, though I did like just skipping pagination on local